cidc_api/models/files/facets.py

```diff
@@ -262,6 +262,10 @@ def __init__(self, match_clauses: List[str], description: Optional[str] = None):
     "Samples Info": FacetConfig(
         ["Clinical Type|Samples Info|samples.csv", "csv|samples info"]
     ),
+    "Clinical data": FacetConfig(
```

Comment:
Maybe something like
```suggestion
    "Trial-Specific Clinical Data": FacetConfig(
```